docs/usage/misc/persistence.md

```diff
@@ -50,3 +50,17 @@ persistence(:influxdb) do
   Item1.average_since(12.hours)
 end
 ```
+
+## Quantity
+
+Most persistence functions (`xxx_since` and `evolution_rate`) return a DecimalType object. This can be converted to Quantity using the `|` operator for comparison against other Quantity based data.
```

Comment:
> When you pull it back out of the persistence, if you persisted it with a quantity type do you get it back out as a quantity or is it always a decimal type?

It is always a DecimalType.

----

However, for example in my example above:
```ruby
grid_draw_avg = Solar_Grid_Power.average_since(15.minutes, :influxdb) | 'W' # The data is in Watt
```
We have the information that the item Solar_Grid_Power is a dimensioned NumberItem, so we could add the code to automatically convert / unitise the return value from Persistence extensions to the corresponding **base** unit of the item. If we did this, `Solar_Grid_Power.average_since()` will return a Quantity. 

This will make the persistence methods somewhat non-consistent, i.e. it can return DecimalType when the item is not dimensioned, and return Quantity when the item is dimensioned.

I quite like this idea.

WDYT?

Reply:
> We have the information that the item Solar_Grid_Power is a dimensioned NumberItem, so we could add the code to automatically convert / unitise the return value from Persistence extensions to the corresponding **base** unit of the item. If we did this, `Solar_Grid_Power.average_since()` will return a Quantity.

I've submitted #178 to show how this works

Reply:
It makes sense to me.. You get back the unit/object you expect based on its type.